logging/logging.go

```diff
@@ -15,8 +16,22 @@ const (
 	jsonLogFormat    = "json"
 )
 
-// CreateLogger create and configure zerolog logger based on app config.
-func CreateLogger(instanceName, format, level string, logOrigin bool) (*zerolog.Logger, error) {
+// GetDefaultLogger create and configure default zerolog logger. It should be used before config is loaded
+func GetDefaultLogger() zerolog.Logger {
+	logger, err := createLogger("spv-wallet-default", jsonLogFormat, "debug", true)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Perhaps, instead of panic - we could return an error and handle it in `main`.
That would be consistent for `main`.

Reply:
we're handling errors by logging it with use of logger,
so if in the first step we're not able to create a logger, it is better to panic,
beside that, this err shouldn't ever happen, because it can only happen if some programmer will use wrong (unknown) log level, so for me, such a mistakes should can be handled with panic.